index.html

```diff
@@ -0,0 +1,201 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Treloo</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header>
+        <h1>Treloo</h1>
+        <nav class="navbar_top">
+            <ul>
+                <li><a href="">Review</a></li>
+                <li><a href="">Tips</a></li>
+                <li><a href="">Alerts</a></li>
+                <li><a href="">Blog</a></li>
+            </ul>
+            <button>Blog</button>
+        </nav>
+        <div class="header_slogan">
+            <h2>Discover Your Life, Travel Where You Want</h2>
+            <p class="slogan">Would you explore natur paradise in the world, let’s find the best destination in world with us.</p>
+        </div>
+    </header>
+    <div class="search">
+        <span class="buttons">
+            <button><img src="Images/logos/Airplane Mode On.png" alt="logo">Flight</button>
+            <button><img src="Images/logos/Hotel Star.png" alt="logo">Hotel</button>
+        </span>
+        <div class="forms">
+            <form class="airports" action="" method="get">
+                <label for="living_from">Living From</label>
+                <input list="city_from" name="living_from" id="living_from">
+                <datalist id="city_from">
+                    <option value="Dublin">
+                    <option value="London">
+                    <option value="Amsterdam">
+                    <option value="Berlin">
+                    <option value="Kharkiv">
+                </datalist>
+                <img src="Images/logos/gray plane.png" alt="logo">
+                <label for="going_to">Going to</label>
+                <input list="city_to" name="going_to" id="going_to">
```

Comment:
There are two option to use label in pair with input.
<label>
    input
</label>
When label contain input - easier to read and understand that input works with label together.
I'd recommend to use that option.

Reply:
done